CriteoPublisherSdk/Sources/Configuration/CR_Config.m

```diff
@@ -27,19 +27,6 @@
 NSString *const CR_ConfigAppEventsUrl = @"https://gum.criteo.com/appevent/v1";
 NSString *const CR_ConfigConfigurationUrl = @"https://pub-sdk-cfg.criteo.com/v2.0/api/config";
 
-// FIXME EE-1001
-// Test (using Pre-Production or Local)
-// #define HIT_LOCAL_CDB // Uncomment to use local CDB instead of preprod
-#ifdef HIT_LOCAL_CDB
```

Comment:
Not needed anymore? You prefer manually editing the URL, or you just don't see the need for using a local CDB?

Reply:
I never had to use this, which is anyway on the production side code to be removed. Yes this is as simple as editing an URL on test side, and if needed it should deserve some dependencyProvider builder method, or, as you suggested a rework on the configuration code which could be improved. In any cases, FAR imo